gitlint/git.py

```diff
@@ -88,12 +92,14 @@ def modified_files(root, tracked_only=False, commit=None):
                 for filename, mode in modified_file_status)
 
 
-def _modified_files_with_commit(root, commit):
+def _modified_files_with_commit(root, diff_target):
     # Convert to unicode and split
-    status_lines = subprocess.check_output([
-        'git', 'diff-tree', '-r', '--root', '--no-commit-id', '--name-status',
-        commit
-    ]).decode('utf-8').split(os.linesep)
+    command = [
+        'git', 'diff-tree', '-r', '--root', '--no-commit-id', '--name-status'
+    ]
+    command.extend(diff_target.split(' '))
```

Comment:
In which cases is the argument going to have spaces, could you give some examples.